### PR TITLE
Refactor layout

### PR DESCRIPTION
**Pull request by wenche**
Part of  [AB#79312](https://dev.azure.com/EquinorASA/d02c350d-092d-41bf-a812-f05d5d8ad1b0/_workitems/edit/79312)

### CODE REVIEW

**Comment by wenche** [comment]
Some comments for AuthContainer:

- Not the ideal place for the PageWrapper, but just didn't bother to take the fight with Typescript 😅 
- Should we just remove the Container from the file and use a plain div or something instead? Would deprecate the Container component 100%

File: components/AuthContainer/AuthContainer.tsx [hunk 1]
```diff
@@ -2,9 +2,18 @@ import { Typography } from "@equinor/eds-core-react"
 import { signIn, useSession } from "next-auth/react"
 import type { FunctionComponent, PropsWithChildren } from "react"
 import { useIntl } from "react-intl"
+import styled from "styled-components"
 
 import { Container } from "../Container"
-import { Page } from "../Page"
+
+const PageWrapper = styled.div`
```
Comment (aronhoyer) [inline components/AuthContainer/AuthContainer.tsx:9]: why is this here as opposed in `Page`? 🤔

File: pages/assets/[id].tsx [hunk 2]
```diff
@@ -112,13 +112,12 @@ const AssetDetailView: NextPage<AssetDetailProps> = ({
   }
 
   return (
-    <>
+    <Page>
       <main>
         <Head>
           <title>{asset?.name}</title>
         </Head>
```
Comment (aronhoyer) [inline pages/assets/[id].tsx:119]: i think this is a good candidate to put into `<Page />`

**Review by aronhoyer — CHANGES_REQUESTED** [review]

**Comment by aronhoyer** [comment]
> * didn't bother to take the fight with Typescript

what was typescript complaining about this time?

File: pages/assets/[id].tsx (hunk 2 above)
Comment (wenche) [reply pages/assets/[id].tsx:119]: That's a good idea :)

Comment (wenche) [reply pages/assets/[id].tsx:119]: > > didn't bother to take the fight with Typescript
> 
> what was typescript complaining about this time?

It didn't like it when I returned just the {children} and we don't need another wrapper

Comment (aronhoyer) [reply pages/assets/[id].tsx]: oh right, because a `FunctionComponent` has to return a `JSX.Element`

**Review by aronhoyer — APPROVED** [review]
